runtime/common/src/weights/tokens.rs

```diff
@@ -50,28 +50,28 @@ pub struct BasiliskWeight<T>(PhantomData<T>);
 
 impl<T: frame_system::Config> WeightInfo for BasiliskWeight<T> {
 	fn transfer() -> Weight {
-		(66_923_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(4 as Weight))
-			.saturating_add(T::DbWeight::get().writes(3 as Weight))
+		(66_005_000 as Weight)
```

Comment:
Are these extra reads and writes part of our change or is this unrelated orml thing?

Reply:
i believe it is the extra read and write of "set fee asset" handler.

Reply:
Jep, that's why I asked to run the weights, cause I wanted us to see the impact.